src/components/PeopleFilters.tsx

```diff
@@ -33,52 +68,25 @@ export const PeopleFilters = () => {
       <div className="panel-block">
         <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
           <div className="level-left">
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=16"
-            >
-              16
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=17"
-            >
-              17
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=18"
-            >
-              18
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=19"
-            >
-              19
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=20"
-            >
-              20
-            </a>
+            {centuriesArray.map(filter => (
+              <a
+                key={filter}
+                data-cy="century"
+                className={cn('button mr-1', {
+                  'is-info': centuries.includes(filter),
+                })}
+                onClick={() => setCenturies(filter)}
+              >
+                {filter}
+              </a>
+            ))}
```

Comment:
just a little **note**: you are iterating through array of centuries, but for some reason u named them as `filter` instead of for example `century`